app/CreateForm.jsx

```diff
@@ -17,55 +16,78 @@ import {
   specialCharacters,
 } from './Validation';
 import inputField from './Fields/inputField';
-import sendProfile from './Api';
+import { sendProfile, startTest } from './Api';
 import selectField from './Fields/selectField';
 import checkboxField from './Fields/checkboxField';
-import { connectivityOptions, locationOptions, baseURL } from './Constants';
+import {
+  connectivityOptions,
+  locationOptions,
+  baseURL,
+  profileLink,
+} from './Constants';
 
 class CreateForm extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      response: '',
+      status: undefined,
+      response: undefined,
     };
```

Comment:
Why initiate as undefined? Why not just  not define them?

Reply:
The status and the response come from the API call and the status can be either "success" or "error", so I don't really know how to initiate without using "undefined" or "null".

Reply:
And the response can be either an error text or on success generated profile link